coinstacks/bitcoin/api/src/websocket.ts

```diff
@@ -0,0 +1,83 @@
+import WebSocket from 'isomorphic-ws'
+import { ErrorResponse, RequestPayload, Topics, TxsTopicData } from '@shapeshiftoss/common-api'
+import { SequencedBTCParseTx } from '@shapeshiftoss/bitcoin-ingester'
```

Comment:
If you're only importing types and no code, you can use `import type`
```suggestion
import type { ErrorResponse, RequestPayload, Topics, TxsTopicData } from '@shapeshiftoss/common-api'
import type { SequencedBTCParseTx } from '@shapeshiftoss/bitcoin-ingester'
```